content/syllabuses/data-eng-part-1.md

```diff
@@ -20,10 +20,12 @@ _db_id: 4
 - {{% contentlink path="projects/tdd/password-checker/part1" flavour="python" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
+- {{% contentlink path="topics/solo-learn/python/8-oop" %}}
 - {{% contentlink path="projects/oop/animals/part2" flavour="python" %}}
 - - {{% contentlink path="specific-skill-success-criteria/classes-and-objects" flavour="python" %}}
 
 - {{% contentlink path="projects/python-specific/file-io" flavour="python" %}}
 - {{% contentlink path="language-agnostic/validate-id-number" flavour="python" %}}
 - {{% contentlink path="sql/shop-project" flavour="" %}}
 - {{% contentlink path="projects/understanding-loops" flavour="python" %}}
+- {{% contentlink path="topics/solo-learn/python/10-pythonicness-and-packaging" %}}
```

Comment:
Wasnt super sure where this one should go so I added it to the end of data eng part 1, please let me know if it should be somewhere else